plugins/jj/jj.plugin.zsh

```diff
@@ -44,11 +54,17 @@ alias jjgcl='jj git clone'
 alias jjgf='jj git fetch'
 alias jjgfa='jj git fetch --all-remotes'
 alias jjgp='jj git push'
+alias jjgpa='jj git push --all'
+alias jjgpd='jj git push --deleted'
+alias jjgpm='jj git push -b main'
+alias jjgpt='jj git push --tracked'
 alias jjl='jj log'
 alias jjla='jj log -r "all()"'
 alias jjn='jj new'
 alias jjrb='jj rebase'
+alias jjrbm='jj rebase -d main'
```

Comment:
This was missed in the README

(Not a blocker to merge IMO as it improves things rather than makes them worse), but this alias and the push alias could be a problem due in repos that still use `master` or other branch names. The git plugin solves this by heuristically checking for a variety of branch names:

https://github.com/ohmyzsh/ohmyzsh/blob/3e7ef0182f59c7990a52cf6ec2981adb56d5b368/plugins/git/git.plugin.zsh#L34-L48

Someone motivated to solve this might want to do something similar. Though it might be easier in jj as we can define a `trunk()` alias and use that? I'm fairly new to jj, so perhaps this should be `jj rebase -d trunk()`

Reply:
Thanks for spotting the missing `jjrbm` docs.
Also I agree that using `trunk()` is better as it is already [configured to point to the right commit](https://jj-vcs.github.io/jj/latest/revsets/#built-in-aliases)